src/components/spreadsheet-view/README.md

```
@@ -0,0 +1,368 @@
+    │   ├── NodesConfigButton                  — node alias management
+    │   ├── SpreadsheetGlobalFilter            — equipment filter selector
+    │   ├── RowCounter                         — filtered row counter
+    │   ├── SaveSpreadsheetButton              — export model / collection
+    │   └── ColumnCreationButton               — add custom column
+    └── SpreadsheetContent (spreadsheet-content/)
+        └── EquipmentTable                     — AG Grid with custom columns, filters, pinned rows
+```
+
+### Data Flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)